setup.cfg

```diff
@@ -1,8 +1,41 @@
-[metadata]
-description-file = README.md
-
 [flake8]
 max-line-length = 100
 application-import-names = autofocus
 import-order-style = appnexus
-exclude = **/__init__.py
+exclude =
+    tests/conftest.py,
+    tests/__init__.py,
+    autofocus/__init__.py,
+    setup.py
+ignore =
+    # https://pep257.readthedocs.io/en/latest/error_codes.html
+    # Missing docstrings in certain cases
+    D100
+    D103
+    D104
+    D107
```

Comment:
this works?? I swear I thought you always had to use commas in these blocks, like:

```
ignore =
    D100,
    D103,
    ...
```

I also never knew you could have lines with commas or empty lines. This is so clean! Def gonna steal it for some of my projects.